parts/kubernetesparams.t

```diff
@@ -69,6 +69,18 @@
       },
       "type": "string"
     },
+    "kubeDnsServiceIP": {
```

Comment:
Similar comment above. Should we add defaults here as a fail safe? 

https://github.com/Azure/acs-engine/blob/master/parts/kubernetesparams.t#L66